### PR TITLE
Update AssetDetails mobile layout

### LINKED ISSUE #261: Asset Detail Screen: Add Mobile layout
It seems to be broken ...

Screenshot from 2020-07-10 10-43-56

### PR DESCRIPTION
**Pull request by FrozenIce0617**
Closes #261 

- [x] Update AssetDetails view for Mobile Layout
- [x] Update TransationTable for Mobile Layout

### CODE REVIEW

**Comment by thatStrangeGuyThorchain** [comment]
nice.
But there is no any way to scroll page on mobile device.
image

File: src/components/wallet/UserTransactionsTable.tsx [hunk 1]
```diff
@@ -50,31 +50,39 @@ const getColumnsRenderers = (address: string): Record<Column, (value: any, tx: U
...
-        align="left"
-        render={columnsRenderers.address}
-      />
-      <Table.Column
-        title={intl.formatMessage({ id: 'common.to' })}
-        dataIndex="Type"
-        align="left"
-        render={columnsRenderers.to}
-      />
+      {isDesktopView && (
```
Comment (thatStrangeGuyThorchain) [inline src/components/wallet/UserTransactionsTable.tsx:61]: hm. is there any reason to to split all of this views to separate condition renders ? 
i mean there are 3 identical checks for `isDesktop`

**Comment by veado** [comment]
Check the border color [with layouts at Figma](https://www.figma.com/file/LgRqZCGMShZ2mJ702MIqyV/ASGARDEX-2020?node-id=1307%3A607), pls.

Screenshot from 2020-07-10 15-13-47

**Comment by veado** [comment]
@FrozenIce0617 Pls provide a row for the dates.

**Comment by veado** [comment]
Check text color of table headers, pls. They should be `#969DAB` (not `#d1d5da`)

Screenshot from 2020-07-10 15-16-03

File: src/components/wallet/UserTransactionsTable.tsx (hunk 1 above)
Comment (veado) [reply src/components/wallet/UserTransactionsTable.tsx:61]: @FrozenIce0617 _As a suggestion:_ Similar to `PoolsOverview` you can split columns into `mobileColumns` and `desktopColumns` . See https://github.com/thorchain/asgardex-electron/blob/32e7296709c9a1d6808b54f6635b920a43e292f3/src/views/pools/PoolsOverview.tsx#L290-L302

File: src/types/wallet/index.ts [hunk 2]
```diff
@@ -48,6 +48,7 @@ export type TransactionPartyType = {
   msg: string
   label: string
   address?: string | null
+  timeStamp?: string | null
```
Comment (veado) [inline src/types/wallet/index.ts:51]: No need to type it as `null`. `?` makes it optional (`undefined`). Same for `address`.

**Comment by veado** [comment]
> [#263 (comment)](https://github.com/thorchain/asgardex-electron/pull/263#issuecomment-656671072)
> ^ We do need a row (not a column) for dates to group transactions by date. Thank you!

Will be implemented by https://github.com/thorchain/asgardex-electron/issues/277

File: src/types/wallet/index.ts (hunk 2 above)
Comment (veado) [reply src/types/wallet/index.ts:51]: Will be fixed with #259

**Pull request merged**